io.c

```diff
@@ -9068,19 +9068,19 @@ rb_io_set_encoding_by_bom(VALUE io)
  *    File.new('t.txt')
  *
  *  Optional argument +mode+ (defaults to 'r') must specify a valid mode
- *  see {\IO Modes}[#class-IO-label-Modes]:
+ *  see {\IO Modes}[IO.html#class-IO-label-Modes]:
  *
  *    File.new('t.tmp', 'w')
  *    File.new('t.tmp', File::RDONLY)
  *
  *  Optional argument +perm+ (defaults to 0666) must specify valid permissions
- *  see {File Permissions}[#class-File-label-Permissions]:
+ *  see {File Permissions}[#class-File-label-File+Permissions]:
```

Comment:
Also this kind of error won't occur.
```suggestion
 *  see File@Permissions:
```